crates/sui-framework/packages/sui-system/sources/sui_system.move

```diff
@@ -77,6 +77,9 @@ module sui_system::sui_system {
     const ENotSystemAddress: u64 = 0;
     const EWrongInnerVersion: u64 = 1;
 
+    // First protocol version in which the AuthenticatorState system object is used.
+    const AuthenticatorStateFirstProtocolVersion: u64 = 23;
```

Comment:
Is this used for anything? `const`s are private, so if it's not, then you may risk getting warnings about unused elements in the framework, in future.

Reply:
no, this is a left over from a previous PR, deleting